interbotix_ros_xsarms/interbotix_xsarm_control/launch/xsarm_control.launch.py

```diff
@@ -0,0 +1,293 @@
+            "robot_name:=",
+            robot_name,
+            " ",
+            "base_link_frame:=",
+            base_link_frame,
+            " ",
+            "show_ar_tag:=",
+            show_ar_tag,
+            " ",
+            "show_gripper_bar:=",
+            show_gripper_bar,
+            " ",
+            "show_gripper_fingers:=",
+            show_gripper_fingers,
+            " ",
+            "use_world_frame:=",
+            use_world_frame,
+            " ",
+            "external_urdf_loc:=",
+            external_urdf_loc,
+            " ",
+            "load_gazebo_configs:=",
+            load_gazebo_configs,
+        ]
+    )
+
+    xsarm_description_launch = IncludeLaunchDescription(
+        PythonLaunchDescriptionSource(
+            [
+                PathJoinSubstitution(
+                    [description_prefix, "launch", "xsarm_description.launch.py"]
+                )
+            ]
+        ),
+        launch_arguments={
+            "robot_model": robot_model,
+            "robot_name": robot_name,
+            "base_link_frame": base_link_frame,
+            "show_ar_tag": show_ar_tag,
+            "show_gripper_bar": show_gripper_bar,
+            "show_gripper_fingers": show_gripper_fingers,
+            "use_world_frame": use_world_frame,
+            "external_urdf_loc": external_urdf_loc,
+            "use_rviz": use_rviz,
+        }.items(),
+    )
+
+    xs_sdk_node = Node(
+        condition=UnlessCondition(use_sim),
+        package="interbotix_xs_sdk",
+        executable="xs_sdk",
+        name="xs_sdk",
+        namespace=robot_name,
+        arguments=[],
+        parameters=[
+            {
+                "motor_configs": motor_configs,
+                "mode_configs": mode_configs,
+                "load_configs": load_configs,
+                "robot_description": model,
+            }
+        ],
+        output="screen",
+    )
+    xs_sdk_sim_node = Node(
+        condition=IfCondition(use_sim),
+        package="interbotix_xs_sdk",
+        executable="xs_sdk_sim",
+        name="xs_sdk_sim",
+        namespace=robot_name,
+        arguments=[],
+        parameters=[
+            {
+                "motor_configs": motor_configs,
+                "mode_configs": mode_configs,
+                "robot_description": model,
+            }
+        ],
+        output="screen",
+    )
+
+    return [
+        xs_sdk_node,
+        xs_sdk_sim_node,
+        xsarm_description_launch,
+    ]
+
+
+def generate_launch_description():
+    return LaunchDescription(
+        [
+            DeclareLaunchArgument(
+                "robot_model",
+                default_value=TextSubstitution(text=""),
+                description=(
+                    "model type of the Interbotix Arm such as 'wx200' or 'rx150'"
+                ),
+            ),
+            DeclareLaunchArgument(
+                "robot_name",
+                default_value=LaunchConfiguration("robot_model"),
```

Comment:
This line makes `robot_name` default to `robot_model` if not specified, similar to the behavior of the ROS1 version of this package.